src/node/util.ts

```diff
@@ -458,17 +458,11 @@ enum CharCode {
  * Taken from vs/base/common/uri.ts. It's not imported to avoid also importing
  * everything that file imports.
  */
-export function pathToFsPath(path: string | string[], keepDriveLetterCasing = false): string {
+export function pathToFsPath(path: string, keepDriveLetterCasing = false): string {
   const isWindows = process.platform === "win32"
-  const uri = { authority: undefined, path: getFirstString(path), scheme: "file" }
+  const uri = { authority: undefined, path: getFirstString(path) || "", scheme: "file" }
```

Comment:
just wondering - would this be more readable in expanded form, e.g.

```typescript
const uri = {
  authority: undefined,
  path: getFirstString(path) || "",
  scheme: "file",
}
```

I'm not sure if prettier prefers this form though

Reply:
Definitely is for me! I think Prettier autoformats it though :(